stackprinter/utils.py

```diff
@@ -11,7 +11,13 @@ def match(string, patterns):
         patterns = [patterns]
     elif patterns is None:
         return False
-    return any([bool(re.search(p, string)) for p in patterns])
```

Comment:
re.search already accepts a compiled pattern, can you demonstrate the above not working? Maybe the problem is that you passed a single pattern instead of a list, in which case you'd need to change code a bit further up.